reactive/flume_hdfs.py

```diff
@@ -0,0 +1,42 @@
+from charms.reactive import when, when_not
+from charms.reactive import set_state, remove_state
+from charmhelpers.core import hookenv
+from charms.flume import Flume
+from charms.hadoop import get_dist_config
```

Comment:
Gah.  You're right.  I'm just going to move it into the class in the base layer to reduce boilerplate and chances for these mistakes.